content/applications/finance/accounting/others/adviser/budget.rst

```diff
@@ -1,157 +1,101 @@
 
-If you record a transaction that has an analytic account assigned to it that *is*
-included in a budget line but one of the general ledger accounts *is not* included
-in the budgetary position for that same budget line, it will not appear within the
-'practical amount' column of that budget line.
+To define budgetary positions, go to :menuselection:`Configuration --> Management: Budgetary
+Positions` and :guilabel:`New`. Add a :guilabel:`Name` to your budgetary position and select the
+:guilabel:`Company` it applies to. Click :guilabel:`Add a line` to add one or more accounts.
 
-Some budgetary positions might be already installed with your chart of
-accounts.
+.. note::
+   Each budgetary position can have any number of accounts from the chart of accounts, though it
+   must have at least one.
 
-To define the positions enter the :menuselection:`Accounting module --> Configuration -->
-Budgetary Positions`.
+Use case
+========
 
-For our example we need to define what accounts relates to our project's
-expenses. Create a position and add items to select the accounts.
+Let’s illustrate this with an example.
 
-.. image:: budget/budget03.png
-   :align: center
+We just started a project with *Smith & Co*, and we would like to budget the income and expenses of
+that project. We plan on having a revenue of 1000, and we don’t want to spend more than 700.
 
-In this case we select the three relevant accounts used wherein we will
-book our expenses.
+First, we need to define what accounts relate to our project’s expenses. Go to
+:menuselection:`Configuration --> Management: Budgetary positions`, and click :guilabel:`New` to add
+a position. Add the accounts wherein expenses will be booked.
 
-.. image:: budget/budget04.png
+.. image::  budget/smith-and-co-expenses.png
    :align: center
+   :alt: display the Smith and Co expenses
 
-Click on *Select*.
+Let's repeat the steps to create a budgetary position that reflects the revenue.
 
-.. image:: budget/budget05.png
+.. image::  budget/smith-and-co-revenue.png
    :align: center
+   :alt: display the Smith and Co revenue
 
-Save the changes to confirm your Budgetary position.
+Analytical accounts
+-------------------
 
-Repeat this steps to create a revenue budgetary position. Only in this
-case select the relevant income accounts.
+Odoo needs to know which costs or expenses are relevant to a specified budget, as the above general
+accounts may be used for different projects. Go to :menuselection:`Configuration --> Analytic
+Accounting: Analytic Accounts` and click :guilabel:`New` to add a new **Analytic Account** called
+*Smith & Co*.
 
-Analytical account
-------------------
+The :guilabel:`Plan` field has to be completed. Plans group multiple analytic accounts, they are
+used to distribute costs and benefits to analyse business performance. :guilabel:`Analytic Plans`
+can be created or configured by going to :menuselection:`Configuration --> Analytic Accounting:
+Analytic Plans`.
```

Comment:
l60: this should be divided either with ";" or "."
l61: just a suggestion: "are used to" can be removed, analyze (American en), Analytical plans is the concept, not something on-sceen so "**"
l62: don't forget the app name in menuselection except if it is part of a flow (not the case here) --> please check all menuselection because I've seen it before as well
```suggestion
The :guilabel:`Plan` field has to be completed. Plans group multiple analytic accounts; they
distribute costs and benefits to analyze business performance. **Analytic Plans** can be
created or configured by going to :menuselection:`Accounting --> Configuration --> Analytic
Accounting: Analytic Plans`.
```